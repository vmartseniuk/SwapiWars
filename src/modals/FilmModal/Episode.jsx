/* eslint-disable react/prop-types */
/* eslint-disable react/jsx-one-expression-per-line */
import React, { memo } from "react";

import { makeStyles, Typography } from "@material-ui/core";
import convertToRoman from "utils/toRoman";

const useStyles = makeStyles((theme) => ({
  episode: {
    fontSize: 16,
    lineHeight: "24px",
    color: "#726A6A",
    letterSpacing: "0.27em",
  },
}));
const Episode = (props) => {
  const classes = useStyles();
  const { children } = props;
  return <Typography className={classes.episode}>{convertToRoman(children)} episode</Typography>;
};

export default memo(Episode);
