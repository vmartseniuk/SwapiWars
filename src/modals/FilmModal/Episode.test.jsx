import { create } from "react-test-renderer";
import Episode from "./Episode";

describe("FilmModal/Episode", () => {
  it("renders correctly", () => {
    const rendered = create(<Episode>1</Episode>);
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
