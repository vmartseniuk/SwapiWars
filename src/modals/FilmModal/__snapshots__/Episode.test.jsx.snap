// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmModal/Episode renders correctly 1`] = `
<p
  className="MuiTypography-root makeStyles-episode-1 MuiTypography-body1"
>
  I
   episode
</p>
`;
